Makefile

```diff
@@ -75,6 +75,6 @@ storybook-deploy:
 	$(STORYBOOK_TO_PAGES)
 
 
-.PHONY: release-major release-minor release-patch
-release-major release-minor release-patch: build
-	@$(XYZ) --increment $(@:release-%=%) --branch $(RELEASE_BRANCH)
+.PHONY: release-major release-minor release-patch release-premajor release-preminor release-prepatch release-prerelease
+release-major release-minor release-patch release-premajor release-preminor release-prepatch release-prerelease: build
+	@$(XYZ) --increment $(@:release-%=%) --branch $(RELEASE_BRANCH) --prerelease-label beta
```

Comment:
is the `--prerelease-label` argument ignored if you specify a non-prerelease increment? just making sure this doesn't make everything release under `beta`

Reply:
_yeah it'll only get applied if using a `pre*` version https://github.com/davidchambers/xyz/blob/bdbe35b680205d6295e716dcc0f824abad54c6a9/xyz#L178